### DISCUSSION TITLE
Toml Jinja filter

### DISCUSSION THREAD

**Discussion by ghost** [saltstack/salt#67411] [Feature Requests]
### Description of Issue/Question

Lots of software, the TICK stack in particular, uses Toml for configuration files. Writing pillar data in Toml is quite difficult in Salt and as far as I'm aware requires writing your own Toml encoder in Jinja using macros. It would be great if there was a Toml filter, like we have JSON and Yaml filters, so we could do:

    {{ config | toml }}

to write a Toml configuration file.

**Comment by ghost** [comment]
I was thinking something like this https://github.com/saltstack/salt/pull/41946

**Comment by mchugh19** [comment]
This was merged as a serializer in #45054
